phpBB/viewonline.php

```diff
@@ -263,6 +263,55 @@
 		case 'viewtopic':
 			$forum_id = $row['session_forum_id'];
 
+			if (!$forum_id)
+			{
+				$array_match_index = 5;
```

Comment:
There probably should be a cleaner way than match exactly index of 5.

Reply:
I put the constant there because it applies to several following regexs and it seemed easier to support if the regex gets changed in the future.  alternatives welcome.